code/datums/mutantraces.dm

```diff
@@ -825,6 +826,11 @@
 
 			var/duration = 3000
 			var/datum/ailment_data/disease/D = mob.find_ailment_by_type(/datum/ailment/disease/lycanthropy/)
+
+			if(mob.client)
```

Comment:
Why this check? It doesn't make sense to me and is potentially abusable (logout at the right time toavoid getting the mutations).

Reply:
Forgot to remove that check when moving everything over to a separate mutation. Removed it!